### DISCUSSION TITLE
So cool, thank you author and contributers

### DISCUSSION THREAD

**Discussion by joexue** [wezterm/wezterm#1380] [Show and tell]
I have been looking for a long time to get a terminal on Windows and Linux like an iTerm2 on MacOs. One feature I really like about iTerm2 is the tmux control mode, follow this feature I found this amazing project. I know someone is working on the tmux control mode on this project, but with the native multiplexity function, we even don't need the tmux anymore!. That's a really really good feature.

So happy to find this project.

**Comment by joexue** [comment]
one more thing, if the multiplexing support ET (Eternal terminal) connection, that will be better. ssh connection is not so robust as we work at home.

**Reply by wez** [reply]
You don't need ET for this; you can use either [TLS Domains](https://wezfurlong.org/wezterm/multiplexing.html#tls-domains) or [SSH Domains](https://wezfurlong.org/wezterm/multiplexing.html#ssh-domains) to achieve the same workflow.

**Reply by joexue** [reply]
Hi Wez, yes, I'm using SSH Domain now, the problem is SSH connection is not so stable especially when we work at home. So we frequently need to reconnect to the host and recreate the session. With ET, we don't need to care about that, after the network connection is back, ET will automatically recover all things.

**Reply by joexue** [reply]
say if your VPN connection is lost, the ssh connection will need to reconnect . But with ET, you do nothing, even you close VPN, once you reconnect the VPN, the ET will recover all thing.s

**Reply by wez** [reply]
Use TLS Domains; they use SSH to initiate, but then transparently reconnect after a disconnect.

**Reply by joexue** [reply]
works like charm!!

Can I depress the follow output when the network is disconnected?

**client disconnected Error while decoding response pdu; will reconnect (done).....**
